nix/shell.nix

```diff
@@ -19,11 +19,15 @@ in
     inputs'.cardano-world.cardano.packages.cardano-address
     inputs'.cardano-world.cardano.packages.cardano-node
     inputs'.cardano-world.cardano.packages.cardano-cli
+    # ^^^^^ 2.7G
 
     pkgs.z3
     pkgs.sqitchPg
     pkgs.postgresql
+
     pkgs.haskellPackages.hspec-golden
```

Comment:
Is there some way to pin `hspec-golden` so that is uses the same GHC as the rest of the build?